test/unit/util/hash-project-loader-hoc.test.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ProjectLoaderHOC from '../../../src/lib/project-loader-hoc.jsx';
+import HashParserHOC from '../../../src/lib/hash-parser-hoc.jsx';
+import storage from '../../../src/lib/storage';
+import {mount} from 'enzyme';
+
+jest.mock('react-ga');
+
+describe('Hash/ProjectLoaderHOC', () => {
+    test('when there is no project data, it renders null', () => {
```

Comment:
This seems like a pure `ProjectLoaderHOC` test, so it should probably stay in the other file.